services/metadata/backend/data/delille.json

```diff
@@ -137,8 +137,8 @@
                 {
                     "__type": "URL",
                     "type": "URL",
-                    "url": "https://delille.dasch.swiss/",
-                    "text": "https://delille.dasch.swiss/"
+                    "url":
+                    "text":
```

Comment:
again, not valid JSON.

if the URL must be removed under all circumstances, then the whole link object must be removed. But ideally, this would just point to another page that can be linked, right?